stories/components/login.stories.js

```diff
@@ -28,3 +28,19 @@ export default {
 export const Login = () => html`
   <mgt-login></mgt-login>
 `;
+
+export const Templates = () => html`
+<mgt-login>
+  <template data-type="signed-in-button-content">
+    {{personDetails.givenName}}
+  </template>
+  <template data-type="flyout-commands">
+      <div>
+          <div>
+              <button data-props="{@click: handleSignOut}">Sign Out</button>
+              <button>Go to my profile</button>
+          </div>
+      </div>
```

Comment:
The nested divs seem unnecessary. It works well if you just plop the buttons in the template root.

Reply:
No idea why, good catch. Fixed